haiku-apps/wakeup/wakeup-1.0.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="An application that plays a sound after a given time"
+DESCRIPTION="WakeUp is a timer application that allows users \
+to set their own ringtone, giving them a more personalized \
+experience. This application could be used for many purposes, \
+like an alarm clock or a cooking timer, just to name a few."
+HOMEPAGE="https://github.com/HaikuArchives/WakeUp"
+COPYRIGHT="1999 Jonathan Villemure"
+LICENSE="MIT"
+REVISION="1"
+gitSrcRevision="116afedea73ba9779d3d641028b65fdb2dc19012"
+SOURCE_URI="https://github.com/HaikuArchives/WakeUp/archive/$gitSrcRevision.tar.gz"
+CHECKSUM_SHA256="28eb8504a2a436a117821aaa399b8c37f5363bd9fcb2ff814f6cb1b6264da311"
+SOURCE_FILENAME="WakeUp-$gitSrcRevision.tar.gz"
+SOURCE_DIR="WakeUp-$gitSrcRevision"
+
+ARCHITECTURES="x86_gcc2 x86_64"
```

Comment:
WakeUp builds and works fine on x86_gcc2 but it fails to build on x86_64.
So we need to mark x86_64 as broken (with a "!"):
```diff
-ARCHITECTURES="x86_gcc2 x86_64"
+ARCHITECTURES="x86_gcc2 !x86_64"
```

In situations like this one, if we still had support for x86 hybrid, then we would have added support for building this recipe for the x86_gcc2 secondary arch by adding:
```
SECONDARY_ARCHITECTURES="x86_gcc2"
```
as well as a few $secondaryArchSuffix at some required places.

It would even be usable on x86_64 if that platform gained support fox x86_gcc2 as secondary arch. But it seems this is not going to happen, so the recipe will be ready for merge once you add the "!" in front of the "x86_64" (in ARCHITECTURES).